src/smc-webapp/r_misc/number-input-backend.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ *  This file is part of CoCalc: Copyright © 2020 Sagemath, Inc.
+ *  License: AGPLv3 s.t. "Commons Clause" – see LICENSE.md for details
+ */
+
+import * as React from "react";
+
+interface Props {
+  number: number;
+  unit?: string;
+}
+
+export const NumberInput: React.FC<Props> = (props: Props) => {
```

Comment:
What the heck is this?  It says "Input" but it is clearly not an input.  Confused.